build/ci/evergreen.yml

```diff
@@ -78,6 +78,24 @@ functions:
           E2E_TAGS: e2e,atlas
           <<: *go_options
         command: make e2e-test
+  "atlas logs e2e":
```

Comment:
[q] do you want to do the same for metrics and clusters?
we'll end up with 4 e2e tasks for atlas on evg, one general, one for clusters, one for metrics and one for logs all run on parallel